src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -209,13 +208,13 @@ public bool Expired
         {
             get
             {
-                return (_expires != DateTime.MinValue) && (_expires.ToLocalTime() <= DateTime.Now);
+                return (Expires != DateTime.MinValue) && (Expires.ToLocalTime() <= DateTime.Now);
             }
             set
             {
-                if (value == true)
+                if (value)
                 {
-                    _expires = DateTime.Now;
+                    Expires = DateTime.Now;
```

Comment:
>It's more consistent with guidelines (use a property instead of a backing field where possible)

Where in the "guidelines" did you see this? Using properties adds some additional overhead.